### PR TITLE
[RELAY] bugfix.

### PR DESCRIPTION
**Pull request by srkreddy1238**
Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/dmlc/tvm/blob/master/CONTRIBUTORS.md#reviewers).

### CODE REVIEW

File: tests/python/relay/test_op_level3.py [hunk 1]
```diff
@@ -87,6 +87,13 @@ def test_transpose_infer_type():
     assert yy.checked_type == relay.TensorType(
         (t, n, 100), "float32")
 
+    y = relay.transpose(x)
+    "axes=" in y.astext()
```
Comment (junrushao1994) [inline tests/python/relay/test_op_level3.py]: What does this line mean?

**Comment by junrushao1994** [comment]
I see, so sometimes it is not good to convert a None axes to an empty list. Looks like ‘axes = axes or []” can be buggy when axes is 0 as well. So could you fix other places containing this line? Thanks!

File: tests/python/relay/test_op_level3.py (hunk 1 above)
Comment (jroesch) [reply tests/python/relay/test_op_level3.py]: I believe he is checking that the text formatting code correctly adds the axes argument.

Comment (junrushao1994) [reply tests/python/relay/test_op_level3.py]: Should we add an assert here?

**Comment by srkreddy1238** [comment]
cc @jroesch  @junrushao1994

**Pull request merged**